eth/state_accessor.go

```diff
@@ -92,7 +92,7 @@ func (eth *Ethereum) stateAtBlock(block *types.Block, reexec uint64) (statedb *s
 			return nil, nil, fmt.Errorf("processing block %d failed: %v", block.NumberU64(), err)
 		}
 		// Finalize the state so any modifications are written to the trie
-		root, err := statedb.Commit(eth.blockchain.Config().IsEIP158(block.Number()))
+		root, _, err := statedb.Commit(eth.blockchain.Config().IsEIP158(block.Number()))
```

Comment:
These look right compared against upstream. What's the story with these—did we just resolve the wrong line during the merge?